### ISSUE TITLE
Project fails to compile in openbmc master.

### ISSUE THREAD

**Issue by williamspatrick** [openbmc/obmc-console#2]
```
/tmp/openbmc.ab3ab48696545824b974eea8308c533469c631cc.fftCgjZK/build/tmp/work/core2-64-openbmc-linux/obmc-console/1.0-r1/git/console-server.c:494:2: error: missing initializer for field 'has_arg' of 'const struct option' [-Werror=missing-field-initializers]
  { 0 },
  ^
In file included from /tmp/openbmc.ab3ab48696545824b974eea8308c533469c631cc.fftCgjZK/build/tmp/work/core2-64-openbmc-linux/obmc-console/1.0-r1/git/console-server.c:32:0:
/tmp/openbmc.ab3ab48696545824b974eea8308c533469c631cc.fftCgjZK/build/tmp/sysroots/qemux86-64/usr/include/getopt.h:109:7: note: 'has_arg' declared here
   int has_arg;
```

**Comment by shenki** [comment]
This is fixed by the commit in #1

**Comment by jk-ozlabs** [comment]
Fixed in https://github.com/openbmc/obmc-console/commit/f5858b5bf75ef82dfc15b7284abd2f4b0d8278e4, closing.
